### DISCUSSION TITLE
我们公司有这样一个需求,模型可能会迭代更新,那么我可以定时将数据库中存储的特征值update同步一下,但是milvus中已经插入的特征值怎么同步更新,我没有找到go的update或者修改feature操作,难道删除milvus数据重新插入么?

### DISCUSSION THREAD

**Discussion by computerAItest** [milvus-io/milvus#4350] [Q&A and General discussion] [answered]
我们公司有这样一个需求,模型可能会迭代更新,那么我可以定时将数据库中存储的特征值update同步一下,但是milvus中已经插入的特征值怎么同步更新,我没有知找到go的update操作,难道删除milvus数据重新插入么?

**Comment by shiyu22** [comment, marked as answer]
Hi,

You're right, Milvus doesn't support update vectors right now, you can `delete` and `insert` to do so.

And there is an [issue#3189](https://github.com/milvus-io/milvus/issues/3198) to get this new feature, you can follow it or add your comments.
